app/src/main/java/org/stepic/droid/ui/activities/MainFeedActivity.kt

```diff
@@ -152,6 +165,11 @@ class MainFeedActivity : BackToExitActivityWithSmartLockBase(),
 
         initGoogleApiClient(true)
 
+        // TODO Testing analytics
```

Comment:
commented code

Reply:
For testing purposes, will remove after finishing the feature.

Reply:
still needed?